src/utils/utils.js

```diff
@@ -26,4 +28,29 @@ export function sortArray(sortType, arrayToSort) {
             break;
     };
     return sortedArray;
-};
\ No newline at end of file
+};
+
+export function randomKeys(numberOfProducts) {
+    const randomCount = faker.number.int({ min: 1, max: numberOfProducts });
+    const maxCount = numberOfProducts - 1;
+    const randomKeys = faker.helpers.uniqueArray(() => faker.number.int({ min: 0, max: maxCount}), randomCount );
+    return randomKeys
+}
+
+export async function productInfoByID(id, itemsList) {
```

Comment:
All selectors should be accessible via page object file. Now how could you know which page its related to?

Reply:
now you have 2 pages with similar selectors, but its better to create page fragment with these locators/functionality 
 or duplicate this logic instead of having the method in utils

Reply:
I have moved a function productInfoByID to both pages inventoryPage and shopingCartPage.
I don't see a need to create a fragment at this point. There is only one function on these pages where a fragment like 'inventoryItemInfoFragment' can be reused.